templates/staffDropboxList.hbs

```diff
@@ -1,37 +1,97 @@
 {{> header }}
+<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
+<link href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css" rel="stylesheet" />
+<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js"></script>
+
```

Comment:
These imports shouldn't be here - you'll need to select the applicable bootstrap components [here](https://getbootstrap.com/docs/3.4/customize/) and then update the files in the css and js folders. Imports go in the header template